plugins/versionpress/src/Database/wordpress-schema.yml

```diff
@@ -13,6 +13,8 @@ post:
     #  - Attachments without post_title
     #  - Updating draft via AJAX
     #  - Any changes saved by clicking "Preview"
+  ignored-columns:
+    - comment_count: '@\VersionPress\Synchronizers\PostsSynchronizer::fixCommentCounts'
```

Comment:
I'm not sure about this notation. It looks simple but maybe it can be confusing. In one case is the column name a value, in other case it's a key and value is a function. What do think, @borekb?